mpf/devices/score_reel_group.py

```diff
@@ -67,7 +67,7 @@ async def _initialize(self):
             if self.config['chimes'][i]:
                 if not self.reels[i]:
                     self.raise_config_error("Invalid reel for chime {}".format(self.config['chimes'][i]), 1)
-                self.machine.events.add_handler(event='reel_' + self.reels[i].name + '_advance',
+                self.machine.events.add_handler(event='reel_' + self.reels[i].name + '_advancing',
```

Comment:
We could just keep the old event here as well. That way we keep backward compatibility. Guess we should mark it as deprecated and remove it in some future major version.

Reply:
In the other file like 168, I changed it from advance to advanced.  And if I put both of them there, it will double fire, once for event handler of advancing and again for advance/d.  But you might have a magic way to tell it to only do one or the other.

Maybe I am missing what you are saying, but the event is code generated, we are just moving it to a different code posted event.